From 4ced3926aaea2cb0822cd8b8ca9cf585e00b2c1a Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@apache.org>
Date: Mon, 9 Jul 2012 17:40:31 +0000
Subject: [PATCH 0142/1357] MAPREDUCE-2739. Update installation docs (remove YarnClientFactory) (bowang via tucu)

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/trunk@1359303 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit acf92b37d1adbf3b845dcf9bb4700e2b383a9f49)

Conflicts:

	hadoop-mapreduce-project/CHANGES.txt
---
 hadoop-mapreduce-project/INSTALL |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hadoop-mapreduce-project/INSTALL b/hadoop-mapreduce-project/INSTALL
index cac914b..e35c8ca 100644
--- a/hadoop-mapreduce-project/INSTALL
+++ b/hadoop-mapreduce-project/INSTALL
@@ -64,7 +64,7 @@ Step 12) sbin/mr-jobhistory-daemon.sh start historyserver
 Step 13) You are all set, an example on how to run a mapreduce job is:
 cd $HADOOP_MAPRED_HOME
 ant examples -Dresolvers=internal 
-$HADOOP_COMMON_HOME/bin/hadoop jar $HADOOP_MAPRED_HOME/build/hadoop-mapreduce-examples-*.jar randomwriter -Dmapreduce.job.user.name=$USER -Dmapreduce.clientfactory.class.name=org.apache.hadoop.mapred.YarnClientFactory -Dmapreduce.randomwriter.bytespermap=10000 -Ddfs.blocksize=536870912 -Ddfs.block.size=536870912 -libjars $YARN_HOME/modules/hadoop-mapreduce-client-jobclient-*.jar output 
+$HADOOP_COMMON_HOME/bin/hadoop jar $HADOOP_MAPRED_HOME/build/hadoop-mapreduce-examples-*.jar randomwriter -Dmapreduce.job.user.name=$USER -Dmapreduce.randomwriter.bytespermap=10000 -Ddfs.blocksize=536870912 -Ddfs.block.size=536870912 -libjars $YARN_HOME/modules/hadoop-mapreduce-client-jobclient-*.jar output 
 
 The output on the command line should be almost similar to what you see in the JT/TT setup (Hadoop 0.20/0.21)
 
-- 
1.7.0.4

